pkg/apis/triggers/v1alpha1/event_listener_validation_test.go

```diff
@@ -159,6 +159,17 @@ func TestEventListenerValidate_error(t *testing.T) {
 		el   *v1alpha1.EventListener
 	}{{
 		name: "no triggers",
+		el: &v1alpha1.EventListener{
```

Comment:
Should we add a test case for an invalid EL that has a template field with an empty name?

Reply:
Added.